app/conf/serverConf.go

```diff
@@ -2,20 +2,21 @@ package conf
 
 import "github.com/spf13/viper"
 
-// Config the application's configuration
+// UserConfig the application's configuration
 type ServerConfig struct {
 	ManagerAiPort    int64
 	DaemonApiPort    int64
 	SetupApiPort     int64
 	LatestReleaseAPI string
 	ReleaseAPI string
+	DaemonHeartbeat int64
 }
 
 
 
 
 // LoadUserConfig loads the config from a file
-func LoadServerConfig() (*ServerConfig, error)  {
+func LoadServerConfig() (ServerConfig, error)  {
```

Comment:
another `error` and `err` situation

Reply:
this is defining the return types, error is the types, and err is the var  :)

Reply:
riiiight, that makes a lot of sense.